oracle/datadog_checks/oracle/oracle.py

```diff
@@ -41,7 +44,11 @@ def __init__(self, name, init_config, instances):
         self._user = self.instance.get('username') or self.instance.get('user')
         self._password = self.instance.get('password')
         self._service = self.instance.get('service_name')
+        self._protocol = self.instance.get("protocol", "TCP")
         self._jdbc_driver = self.instance.get('jdbc_driver_path')
+        self._jdbc_truststore_path = self.instance.get('jdbc_truststore_path')
+        self._jdbc_truststore_type = self.instance.get('jdbc_truststore_type')
+        self._jdbc_truststore_password = self.instance.get('jdbc_truststore_password', '')
```

Comment:
does the password need to default to `''`?

Reply:
If I don't set password to `''` by default, connecting to JDBC with an empty password string causes an error. Is there another way I should handle this?